thin_pieces/arc.scad

```diff
@@ -0,0 +1,6 @@
+$fn=200;
```

Comment:
What's the purpose of this test? It seems that it's the same test as thin_cylinder.scad.